cmd/agentbasedinstaller/register.go

```diff
@@ -103,6 +104,7 @@ func RegisterCluster(ctx context.Context, log *log.Logger, bmInventory *client.A
 
 	annotations := aci.GetAnnotations()
 	if installConfigOverrides, ok := annotations[controllers.InstallConfigOverrides]; ok {
+		var reJsonField = regexp.MustCompile(`(?i)"([^"]*(password)[^"_]*)":\s*"[^\,]*"`)
```

Comment:
What's the requirement for the closing underscore `[^"_]`?
For the content part, couldn't be simply everything within quotes?

Reply:
Yeah not sure why I had the underscore, removed it.